app/src/androidTest/java/org/mozilla/fenix/screenshots/DefaultHomeScreenTest.kt

```diff
@@ -30,15 +32,17 @@ class DefaultHomeScreenTest : ScreenshotTest() {
     fun showDefaultHomeScreen() {
         homeScreen {
             verifyAccountsSignInButton()
-            Screengrab.screenshot("HomeScreenRobot_home-screen")
             swipeToBottom()
             Screengrab.screenshot("HomeScreenRobot_home-screen-scroll")
+            TestAssetHelper.waitingTime
         }
     }
 
     @Test
     fun privateBrowsingTest() {
         homeScreen {
+            SystemClock.sleep(TestAssetHelper.waitingTimeShort)
```

Comment:
I know this is not a good practice, but in order to get the correct screen screenshot, I have not found a different way, also this tests don't run on CI, so time is not a critical fact here...